<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordManager</class>
 <widget class="QMainWindow" name="PasswordManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>691</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25, 29, 50);
color:rgb(186, 44, 115)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_account_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>create account</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_username_input">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(40, 47, 68)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_password_input">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(40, 47, 68)</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(186, 44, 115)</string>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="Error">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user_page">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QPushButton" name="add_password_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>Add password</string>
      </property>
     </widget>
     <widget class="QComboBox" name="passwords_combo_box">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(69, 58, 73)</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Platforms</string>
      </property>
     </widget>
     <widget class="QPushButton" name="show_password_btn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>show password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="platform_input">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(40, 47, 68)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Platform</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(40, 47, 68)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_btn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QLabel" name="Error_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="generate_password_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(40, 47, 68)
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>'&lt;RF2X7Yu.l+JoNiAQC$;zsEy:]UH!,dvbGt5Bjc%K[}(=-Ph</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QPushButton" name="generate_password_btn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(109, 59, 71)</string>
      </property>
      <property name="text">
       <string>Generate Password</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="password_length">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>480</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>400</number>
      </property>
      <property name="value">
       <number>16</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="passsword_input">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLabel" name="current_user_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
